Client/mods/deathmatch/logic/CClientVehicle.h

```diff
@@ -530,6 +530,8 @@ class CClientVehicle : public CClientStreamElement
     bool SetDummyPosition(eVehicleDummies dummy, const CVector& position);
     bool ResetDummyPositions();
 
+    void GetEntryPoint(CVector& entryPoint, int entryPointIndex);
```

Comment:
It seems like it would be better to return CVector instead of passing it as an argument.